docx/shape.py

```diff
@@ -101,3 +101,11 @@ def width(self):
     def width(self, cx):
         self._inline.extent.cx = cx
         self._inline.graphic.graphicData.pic.spPr.cx = cx
+
+    @property
+    def alt_text(self):
+        return self._inline.docPr.descr
+
```

Comment:
Maybe this attribute should also be called `descr`?

Reply:
i think that would make sense. that description will later manifest as `alt` attr, but in context of this, it really is image description. agree 👍